score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServicesImpl.java

```diff
@@ -95,11 +96,47 @@ public Long triggerSync(CompilationArtifact compilationArtifact, Map<String, ? e
         slang.unSubscribeOnEvents(scoreEventListener);
         return executionId;
     }
+    @Override
+    public Long triggerSync(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> inputs, Map<String, ? extends Serializable> systemProperties, boolean isQuiet){
```

Comment:
again, I overloaded the method to make sure everything still works before checking it. I will make sure there only one method